tests/Benchmarks/System.Memory/Slice.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Diagnosers;
+
+namespace System.Memory.Benchmarks
+{
+    /// <summary>
+    /// A class containing benchmarks for comparing constructs that do the same task in
+    /// different ways with the same end outcome.
+    /// </summary>
+    // [HardwareCounters(HardwareCounter.InstructionRetired)] // uncomment if you want to get hardware counters
```

Comment:
Can we un-comment this from the get go?

Reply:
sure, but please remember that to get Hardware Counters you need to run benchmarks as Admin (it's ETW limitation, BDN just warns about it)

Reply:
When running other benchmarks, it does not affect the experience.
When running this particular benchmark as non-admin, it's going to complain about it and run it without hardware counters.

So I think that we can keep it.